src/main/scala/rps/Game.scala

```diff
@@ -40,6 +40,14 @@ object Game {
     }
   }
 
+  def printMove(input: Move): String = {
+    input match {
+      case Move.Rock    => "🪨 Rock"
+      case Move.Paper     => "📄 Paper"
+      case Move.Scissors => "✂️ Scissors"
```

Comment:
Forse hai autoformat non funzionante? Mi aspetterei o che le `=>` siano allineate a sinistra (che mi pare sia il default di Scalafmt) oppure che abbiano solo uno spazio dal carattere precedente.

Reply:
Buon occhio. É una cosa che faccio a mano al momento. Sarebbe buono impostare un hook sul save ma non saprei come farlo in un progetto Scala. C'è una impostazione di VSCode che può farlo ma di norma la tengo disabilitata perchè fa casino quando l'autoformat è configurato a livello di progetto e.g. `.prettierrc` o `.editorrc`

Reply:
Noi di solito usiamo il format on save dell'editor (quindi penso l'impostazione che dici) e se esiste una configurazione di progetto dovrebbe in teoria prendere quella. Però non uso VS Code quindi non sono sicuro

(Poi a volte usiamo anche hook, fatti con https://www.npmjs.com/package/husky credo)